tests/e2e/contacts/new-lineage-structure.wdio-spec.js

```diff
@@ -7,12 +6,12 @@ const commonPage = require('../../page-objects/common/common.wdio.page');
 const utils = require('../../utils');
 const sentinelUtils = require('../sentinel/utils');
 
-const centerName = faker.address.city();
-const centerContact = faker.name.findName();
-const area = faker.address.city();
-const areaContact = faker.name.findName();
-const household = faker.address.city();
-const householdContact = faker.name.findName();
+const centerName = 'Franklin';
```

Comment:
We have a GitHub issue tracking the search fix, can we reference it here as a comment to indicate we need to come back and use faker again once that issue is resolved

Reply:
Based on our discussion during standup. We are going to remove faker and work on implementing a secondary suite to run with mocked data.